### DISCUSSION TITLE
Could this be supported with a background color brush type?

### DISCUSSION THREAD

**Discussion by lukewire129** [taublast/DrawnUi#3] [Q&A] [answered]
I can't use the LinearGradientBrush type because I only specify it as BackgroundColor(Color). 

LinearGradientBrush type because I can't set the LinearGradientBrush type. 
support is possible?

![image](https://github.com/taublast/DrawnUi.Maui/assets/54387261/b5a41f9a-de3b-4a9b-af12-e6b001ded79e)

**Comment by taublast** [comment, marked as answer]
Hi.
Actually, the properties SkiaShape is using are not WPF-inspired brushes but rather those inspired by Figma, to allow its designs to be easily recreated:
So you have properties:
* BackgroundColor
* FillGradient
* StrokeGradient (+StrokeWidth, StrokeColor)

So looking at your code it can rewritten:

 ```
     BackgroundColor="Black" whatever
    <draw:SkiaShape.FillGradient>
         <draw:SkiaGradient
             EndXRatio="1"
             EndYRatio="0"
             StartXRatio="0"
             StartYRatio="1"
             Type="Linear">
             <draw:SkiaGradient.Colors>
                 <Color>#0A5125</Color>
                 <Color>#00FFFF</Color>
                 <Color>#00FF00</Color>
             </draw:SkiaGradient.Colors>
             <draw:SkiaGradient.ColorPositions>
                 <x:Double>0.1</x:Double>
                 <x:Double>0.5</x:Double>
                 <x:Double>1.0</x:Double>
             </draw:SkiaGradient.ColorPositions>
         </draw:SkiaGradient>
     </draw:SkiaShape.FillGradient>
```

It is not excluded at all that we start using WPF-like brushes instead of this approach at some point, just need time to evaluate all the pros and cons.

**Comment by lukewire129** [comment]
Thank you!

**Comment by taublast** [comment]
@lukewire129 Some time has passed, MAUI brushes are now supported, latest nugget 1.2.4.8:
https://github.com/taublast/DrawnUi.Maui/issues/110#issuecomment-2454539934
